### PR TITLE
Add `event` example

### PR DESCRIPTION
**Pull request by jayz22**
### What

[TODO: Short statement about what is changing.]

### Why

[TODO: Why this change is being made. Include any context required to understand the why.]

### Known limitations

[TODO or N/A]

### CODE REVIEW

File: event_hello/src/test.rs [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+use super::*;
+use soroban_sdk::{BytesN, Env};
+
+#[test]
+fn test() {
+    let env = Env::default();
+    let contract_id = BytesN::from_array(&env, &[0; 32]);
+    env.register_contract(&contract_id, EventContract);
+    EventContractClient::new(&env, &contract_id).hello(&Symbol::from_str("SourBun"));
```
Comment (leighmcculloch) [inline event_hello/src/test.rs:11]: Is there a way to assert on the events at all? If not no worries, although we should create an issue for that.

**Review by leighmcculloch — COMMENTED** [review]
Thanks for adding this! Couple of small asks inline.

File: event_hello/src/test.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jayz22) [reply event_hello/src/test.rs]: Fixed.

Comment (jayz22) [reply event_hello/src/test.rs]: There isn't. It either successes returning empty or traps on failure. What kind of assertion do you have in mind?

File: event_hello/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+use soroban_sdk::{contractimpl, map, Env, Symbol};
+
+pub struct EventContract;
+
+#[contractimpl]
+impl EventContract {
+    pub fn hello(env: Env, to: Symbol) -> () {
+        const GREETING: Symbol = Symbol::from_str("Hello");
+        let event = env.events();
+        let topics = (GREETING, to);
```
[the remark this answers is not in the document]
Comment (jayz22) [reply event_hello/src/lib.rs]: Got ya. Fixed.

File: event_hello/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+doctest = false
+
+[features]
+default = ["export"]
+export = []
```
[the remark this answers is not in the document]
Comment (jayz22) [reply event_hello/Cargo.toml]: Fixed.

File: event_hello/src/test.rs (hunk 1 above)
Comment (leighmcculloch) [reply event_hello/src/test.rs]: We just need to be able to assert on an event having been published. This isn't in the SDK yet though, so nothing to do here. I opened an issue about it: https://github.com/stellar/rs-soroban-sdk/issues/532.

**Review by leighmcculloch — APPROVED** [review]

**Pull request merged**